docs/source/deep-learning/pytorch/quickstart/pytorch_quickstart.ipynb

```diff
@@ -0,0 +1,1361 @@
+   "source": [
+    "## Prepare the Data\n",
+    "\n",
+    "Let's load our training data `FashionMNIST` from `torchvision`, which has already been preprocessed into scale [0, 1). We then wrap the dataset into an instance of `torch.utils.data.Dataloader`."
+   ],
+   "metadata": {
+    "id": "kIqWWgO72Bwp"
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "training_data = datasets.FashionMNIST(\n",
+    "    root=\"data\",\n",
+    "    train=True,\n",
+    "    download=True,\n",
+    "    transform=ToTensor(),\n",
+    ")\n",
+    "\n",
+    "test_data = datasets.FashionMNIST(\n",
+    "    root=\"data\",\n",
+    "    train=False,\n",
+    "    download=True,\n",
+    "    transform=ToTensor(),\n",
+    ")"
+   ],
+   "metadata": {
+    "id": "zHLPgjFyzw_P"
+   },
+   "execution_count": 3,
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Let's look into our data."
+   ],
+   "metadata": {
+    "id": "l4Z8bDxI3TAh"
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "print(f\"Image size: {training_data[0][0].shape}\")\n",
+    "print(f\"Size of training dataset: {len(training_data)}\")\n",
+    "print(f\"Size of test dataset: {len(test_data)}\")"
+   ],
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "GlZkxQHu2t7V",
+    "outputId": "2ae9190a-03ac-4371-d6a9-0e9309a4e9de"
+   },
+   "execution_count": 4,
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Image size: torch.Size([1, 28, 28])\n",
+      "Size of training dataset: 60000\n",
+      "Size of test dataset: 10000\n"
+     ]
+    }
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "We wrap the dataset into `Dataloader` for batching purposes. `Dataloader` is a useful tool for data preprocessing, for more details, you can refer to the [developer guide from PyTorch](https://pytorch.org/tutorials/beginner/basics/data_tutorial.html#preparing-your-data-for-training-with-dataloaders)."
+   ],
+   "metadata": {
+    "id": "NhV1htwx3qJR"
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "train_dataloader = DataLoader(training_data, batch_size=64)\n",
+    "test_dataloader = DataLoader(test_data, batch_size=64)"
+   ],
+   "metadata": {
+    "id": "RoZj28jT3N_n"
+   },
+   "execution_count": 5,
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Define our Model\n",
+    "\n",
+    "Now let's define our model, we will build a simple convolutional neural network as the classifier. To define a PyTorch model, you will need to subclass from `torch.nn.Module` and override `__init__` to define model components, and `forward()` method to implementation the forward-pass logic.\n",
+    "\n",
+    "We use two convolutional layer and the model output will be the logits of each class (10 classes in total). Applying softmax on logits yields the probability distribution across classes."
```

Comment:
We may want to explain this a bit more. This went from 0 to 100km/h in less than 1 sentence.

Reply:
good call, I added a little text over it. I kinda treat CNN + logits output as a common sense, but it's not true I guess.